src/error.rs

```diff
@@ -62,12 +69,20 @@ impl fmt::Display for TiledError {
                 )
             }
             TiledError::InvalidTileFound => write!(fmt, "Invalid tile found in map being parsed"),
-            TiledError::Other(s) => write!(fmt, "{}", s),
+            TiledError::InvalidEncodingFormat { encoding, compression } => 
+                write!(
+                    fmt,
+                    "Unknown encoding or compression format or invalid combination of both (for tile layers): {} encoding with {} compression",
+                    encoding.as_deref().unwrap_or("no"),
+                    compression.as_deref().unwrap_or("no")
+                ),
+            TiledError::InvalidPropertyValue => write!(fmt, "Found invalid property value"),
+            TiledError::UnknownPropertyType { name } =>
+                write!(fmt, "Found unknown property value type '{}'", name),
```

Comment:
In this and the above case, I would leave out the "Found" from the error message, which seems unnecessary and inconsistent.